src/cosalib/build.py

```diff
@@ -64,7 +65,7 @@ class _Build:
       - _build_artifacts(*args, **kwargs)
     """
 
-    def __init__(self, build_dir, build="latest", workdir=None):
+    def __init__(self, builds_dir, build="latest", workdir=None):
```

Comment:
`build_dir` typo either here or in the rest of the method.

Reply:
Thanks, good catch! Fixed :arrow_down: